susemanager-frontend/susemanager-nodejs-sdk-devel/susemanager-nodejs-sdk-devel.changes

```diff
@@ -1,3 +1,5 @@
+- upgrade serialize-javascript - https://github.com/advisories/GHSA-h9rv-jmmf-4pgx
```

Comment:
nit: maybe add the CVE number addressed as well

Reply:
I had the same thinking but then Julio will complain about the long changelog line. Either we mention the CVE or the link, and inside the link there is the CVE reference. If you agree I'd better keep it as it is. We also did the same in the past changelog lines below in the same file. What do you think?

Reply:
It's less searchable with a URL link but OK, let's keep the lines short and consistency with the past. :shipit:

Reply:
Approved :+1: